### ISSUE TITLE
Bug: integrate(x*sqrt(abs(x)),(x,-1,0)) returns wrong result

### ISSUE THREAD

**Issue by mihaiancu** [sympy/sympy#21549]
```python
from sympy import integrate, sqrt 
from sympy.abc import x 
integrate(x*sqrt(abs(x)), (x, -1, 0))
```

returns  2/5. The correct value is -2/5. The wrong opposite sign result is returned also if we replace x with x**k, where k is a positive integer.

**Comment by vsydoriak99** [comment]
Hello, I checked, everything works correctly.

**Comment by mihaiancu** [comment]
Hello! I am using SymPy 1.8 and I get the wrong result 2/5.  Also, trying the online JupyerLab (https://jupyter.org/try) I get the same wrong result. ``integrate(x*sqrt(abs(x)), (x, -1, 0))`` returns the same values as ``integrate(x*sqrt(abs(x)), (x, 0, 1))``, but in the former case the integrand  is negative.

**Comment by naveensaigit** [comment]
SymPy 1.8 gives `2/5`, but it seems to be fixed on master, which gives `-2/5`

**Comment by oscarbenjamin** [comment]
This was fixed in fe3a75785424dae6b5219b387d12e5c8c1ef0aca from #21404

This issue can be closed if a regression test is added to `test_issue_21166`
